videoframes.py

```diff
@@ -168,6 +180,18 @@ def setup_actions(self):
 
         self.context_menu.addMenu(quality_submenu)
 
+    def open_stream_in_browser(self, event):
+        url = ("http://www." if "www" not in self.stream.url else "http://") + self.stream.url
```

Comment:
Also, an example of this breaking:

<img width="595" alt="screen shot 2017-05-13 at 15 29 53" src="https://cloud.githubusercontent.com/assets/11392811/26025872/1f54b5e2-37f1-11e7-8958-0c229680738f.png">

That is not a working url to open.